themes/default/src/webviz_default_theme/templates/main.html

```diff
@@ -17,6 +17,7 @@
         {% include 'menu.html' %}
         <main class="D(f) Fxd(c) Fxg(1)--sm H(100vh) Ovy(a)--sm Ovs(touch) Scb(s)">
             <div class="Fxg(1) Fxs(0) Mt(80px) Mt(0)--sm Bgc(white)">
+                {% if page.location == webviz.index_page() %}
```

Comment:
Are we sure that only including the banner on the index page is the right behavior?

Reply:
@eivindjahren As long as `banner` is a parameter to the class `Webviz`, it is the expected behaviour to ony get it on the index page. 

The alternative is to remove `banner` from `Webviz`, and make it instead as a `PageElement` (which then can be added on any page). This approach would require to let the different `PageElement`s have varying margin (currently there is [a margin for the whole content](https://github.com/Statoil/webviz/blob/master/themes/default/src/webviz_default_theme/templates/main.html#L29)).

Any preferences?